src/main/kotlin/nl/info/client/brp/BrpClientService.kt

```diff
@@ -24,15 +25,24 @@ import nl.info.client.brp.util.PersonenQueryResponseJsonbDeserializer.Companion.
 import nl.info.client.brp.util.PersonenQueryResponseJsonbDeserializer.Companion.ZOEK_MET_STRAAT_HUISNUMMER_EN_GEMEENTE_VAN_INSCHRIJVING
 import nl.info.zac.util.AllOpen
 import nl.info.zac.util.NoArgConstructor
+import org.eclipse.microprofile.config.inject.ConfigProperty
 import org.eclipse.microprofile.rest.client.inject.RestClient
+import java.util.Optional
 
 @ApplicationScoped
 @AllOpen
 @NoArgConstructor
 class BrpClientService @Inject constructor(
-    @RestClient val personenApi: PersonenApi
+    @RestClient val personenApi: PersonenApi,
+    @ConfigProperty(name = ENV_VAR_BRP_DOELBINDING_ZOEKMET)
+    private val purposeSearch: Optional<String>,
+
+    @ConfigProperty(name = ENV_VAR_BRP_DOELBINDING_RAADPLEEGMET)
+    private val purposeRetrieve: Optional<String>
```

Comment:
```suggestion
    private val retrievePersoonDefaultPurpose: Optional<String>
```
same reason as the other one